2023-04-27 21:35:51,754 INFO    MainThread:1657 [wandb_setup.py:_flush():76] Configure stats pid to 1657
2023-04-27 21:35:51,755 INFO    MainThread:1657 [wandb_setup.py:_flush():76] Loading settings from /Users/malujie/.config/wandb/settings
2023-04-27 21:35:51,755 INFO    MainThread:1657 [wandb_setup.py:_flush():76] Loading settings from /Users/Uni Course/IML/IML_A3/wandb/settings
2023-04-27 21:35:51,755 INFO    MainThread:1657 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'decision_tree_finetuning', 'entity': 'xiaolinzzz', 'root_dir': '/Users/Uni Course/IML/IML_A3'}
2023-04-27 21:35:51,755 INFO    MainThread:1657 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-04-27 21:35:51,756 INFO    MainThread:1657 [wandb_init.py:_log_setup():507] Logging user logs to /Users/Uni Course/IML/IML_A3/wandb/run-20230427_213551-845genmb/logs/debug.log
2023-04-27 21:35:51,756 INFO    MainThread:1657 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/Uni Course/IML/IML_A3/wandb/run-20230427_213551-845genmb/logs/debug-internal.log
2023-04-27 21:35:51,757 INFO    MainThread:1657 [wandb_init.py:init():547] calling init triggers
2023-04-27 21:35:51,757 INFO    MainThread:1657 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-04-27 21:35:51,757 INFO    MainThread:1657 [wandb_init.py:init():595] starting backend
2023-04-27 21:35:51,758 INFO    MainThread:1657 [wandb_init.py:init():599] setting up manager
2023-04-27 21:35:51,768 INFO    MainThread:1657 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-04-27 21:35:51,775 INFO    MainThread:1657 [wandb_init.py:init():605] backend started and connected
2023-04-27 21:35:51,797 INFO    MainThread:1657 [wandb_run.py:_label_probe_notebook():1237] probe notebook
2023-04-27 21:35:51,797 INFO    MainThread:1657 [wandb_run.py:_label_probe_notebook():1247] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-04-27 21:35:51,798 INFO    MainThread:1657 [wandb_init.py:init():695] updated telemetry
2023-04-27 21:35:51,816 INFO    MainThread:1657 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-04-27 21:35:55,023 WARNING MainThread:1657 [wandb_init.py:init():1167] interrupted
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1145, in init
    run = wi.init()
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 735, in init
    result = run_init_handle.wait(
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 574, in wait
    signaled = self._cond.wait(timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
